themes/red-theme/cms/templates/login.html

```diff
@@ -50,8 +51,9 @@ <h1 class="title title-1">${_("Sign In to {studio_name}").format(studio_name=set
 </div>
 </%block>
 
-<%block name="requirejs">
-  require(["js/factories/login"], function(LoginFactory) {
-      LoginFactory("${reverse('homepage')}");
-  });
+<%block name="requirejs_page">
+  <%static:require_page page_name="js/pages/login" class_name="LoginFactory">
+    ## xss-lint: disable=mako-invalid-js-filter, mako-invalid-html-filter
```

Comment:
Will we need to include this disable message for every invocation of `requirejs_page`?

Reply:
Yes, this was my comment about needing to teach the xsslint checker about this kind of tag, so that it expects javascript, rather than html.

Reply:
Turns out it's an easy change, so I've added it as a tag that contains javascript.